public/js/test/mochitest/head.js

```diff
@@ -114,21 +130,30 @@ const initDebugger = Task.async(function* (url, ...sources) {
     win: win
   };
 
-  yield Promise.all(sources.map(url => {
-    return waitForState(dbg, state => {
-      return getSources(state).some(s => s.get("url").includes(url));
-    });
-  }));
+  if(sources.length) {
+    info("Waiting on sources: " + sources.join(", "));
```

Comment:
This might be nice as a separate helper

`return waitForSourcesToLoad(sources)`